src/meta/src/hummock/compaction/compaction_picker.rs

```diff
@@ -189,13 +190,131 @@ impl CompactionPicker for MinOverlappingPicker {
     }
 }
 
+pub struct ManualCompactionPicker {
```

Comment:
This file is too large.... Maybe we shall move them to an independent file

Reply:
fix it , split manual_compaction_filter to indenpendent file